src/main/features/claim/views/amount.njk

```diff
@@ -51,12 +51,12 @@
                     </span>
                   {% endif %}
 
-                  <span class="form-label-bold claim-amount-numbered mobile-show">
+                  <span class="form-label-bold claim-amount-numbered phone-show">
```

Comment:
this should remain as mobile-show as it relates to hiding this when a user in on a mobile